PROJECT

```diff
@@ -1,32 +1,43 @@
 domain: functionmesh.io
-layout: go.kubebuilder.io/v2
+layout:
+- go.kubebuilder.io/v3
 projectName: function-mesh
 repo: github.com/streamnative/function-mesh
 resources:
-- group: compute
+- api:
+    crdVersion: v1
+    namespaced: true
   controller: true
-  domain: streamnative.io
+  domain: functionmesh.io
+  group: compute
   kind: FunctionMesh
-  path: github.com/streamnative/function-mesh/api/compute/v1alpha1
+  path: github.com/streamnative/function-mesh/api/v1alpha1
   version: v1alpha1
-- group: compute
+- api:
+    crdVersion: v1
+    namespaced: true
   controller: true
-  domain: streamnative.io
-  kind: Source
-  path: github.com/streamnative/function-mesh/api/compute/v1alpha1
+  domain: functionmesh.io
+  group: compute
+  kind: Function
+  path: github.com/streamnative/function-mesh/api/v1alpha1
```

Comment:
Just curious here, IIRC, https://github.com/streamnative/function-mesh/pull/473 moves the api from `api/v1alpha1` to `api/compute/v1alpha1`, shouldn't the `path` remains to `github.com/streamnative/function-mesh/api/compute/v1alpha1`? I know this file might not affect the function mesh but seems wired to me.